test/servico/cst02Test.java

```diff
@@ -55,18 +53,22 @@ public void tearDown() {
     public void testeNotificaçãoDeAusência_DiaInteiro() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha(
+                "Funcionario1", "123456"));
         
         ProfessorService professorService = new ProfessorService();
-        ProfessorModel professor = professorService.obterProfessorPorNome("Professor1");
+        ProfessorModel professor = professorService.obterProfessorPorNome(
+                "Professor1");

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "Professor1" appears 4 times in the file.](https://www.codacy.com/app/calebepb/substituicao/file/2777729600/issues/source?bid=3364462&fileBranchId=3364484#l61)